### PR TITLE
NEW_YORK_ARTICLE_PAGE_CLONE

### PR DESCRIPTION
**Pull request by Lidya1234**
This site is the clone of New York Times article. It has three sections the navigation bar, main (with five sections inside it) and a footer. There are links which mimics the real website but they do not function.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+            </div>
+
+
+        </div>
+
+
+
+
+
+
```
Comment (SafaErden) [inline index.html]: Please avoid using redundant empty lines between tags and avoid using extra indentation (2 spaces suggested for each indentation). To provide this, I can suggest you to install the proper code-formatter plugins.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+        <div class="align">
+            <a href="#"><img class="logo" src="images/New-York-Times-Logo.png" alt="image here"></a>
+        </div>
+        <div class="right flex">
+            <button class="btn">SUBSCRIBE NOW</button>
+            <button class="btn">LOG IN</button>
+        </div>
+    </nav>
+
+    <section class="section-1">
```
Comment (SafaErden) [inline index.html]: - [OPTIONAL] Please consider the use of the `<main>` tag for the dominant content of the `<body>` of a document, you can wrap the dominant content with the `<main>` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+                of a second old. They are the long-sought smoking-gun evidence of inflation, proof,
+                Dr. Kovac and his colleagues say, that Dr. Guth was correct.
+            </p>
+
+            <div class="mid">
+                <hr>
+                <h3>Thanks for reading The Times.</h3>
+                <a class="blue" href="#">Subscribe to The Times</a>
+                <hr>
+            </div>
```
Comment (SafaErden) [inline index.html]: - [ ]  Please center the elements to have the same UI with the original page( the content is aligned left in your project):

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,631 @@
...
+                <p>
+                    <strong> EXPANSION </strong> Returning to our coffee, imagine a single, central pot expanding faster
+                    than light and cooling to an even temperature as it expands. That is something like inflation. And
+                    the structure of the universe mirrors the froth and foam of the original pot.
+                </p>
+            </div>
+        </div>
+        <hr>
+        <p>By LARRY BUCHANAN and JONATHAN CORUM</p>
+    </section>
```
Comment (SafaErden) [inline index.html]: - [ ] Please do not use screenshots of the original page as shortcut to bypass requirements. Please create the 'theory of inflation' section by using grid, flex of float both for images and texts (If you want to use a background image like it is used in the original page, then the view should not be broken in different screen sizes like it is in the original page.)

image

File: README.md [hunk 5]
```diff
@@ -1,3 +1,40 @@
 # New York Times Article
 
-First CSS project of Lidya Ghebreigziabher and Abhigyan Mahanta
\ No newline at end of file
+First CSS project of Lidya Ghebreigziabher and Abhigyan Mahanta
+
+<img src="images/projectpage.PNG" alt="NewYork_Times_Article">
+
+Additional description about the project and its features.
```
Comment (SafaErden) [inline README.md]: - [ ] If this line is a title, please make it has the same style with the other titles in your README.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,40 @@
...
+
+This is a project to demonstrate the exisiting knowledge of HTML and CSS. It is a collaborative project which recreates a New York Times article.
+
+<i>Built With HTML5, CSS3, GIT, and VSCODE</i>
+
+Live Demo : https://new-york-article.netlify.app/
+
+<h1>Getting Started</h1>
+
+To get a local copy up and running follow these simple example steps.
```
Comment (SafaErden) [inline README.md:17]: - [ ] I couldn't see the simple steps that you are talking about 😄

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, your project is awesome 😄 it really looks exactly like the original. It has both nav and footer in a fixed position 👐 , However, we need a little fix before we can proceed. Kindly find comments below:

<hr>

- [ ] Your page should have an expandable footer with fixed position (it sticks to the bottom part of the screen, you may not see it if you are logged in) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

image

- [ ] Kindly remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. ([Check this link form better understanding Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/))

**_NOTES:_**

- Your PR should have a meaningful title and a brief description. In this case, your PR does not have a meaningful title. Please consider this issue for your upcoming projects.

<hr>

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

<hr>

Kindly make these changes and submit another code review 👌
Happy coding_ 👨‍💻

**Comment by ikraamg** [comment]
### **Status: PROJECT APPROVED**  :1st_place_medal: :1st_place_medal: :1st_place_medal:

Hi @Lidya1234  and @Abhigyan001  :smile:  Congratulations, your project looks really good and you did an amazing job making the necessary changes :sunglasses:. 

Before you merge and submit your project, please change your live link. It does not reflect your latest changes. 

_**Happy coding and good luck on the next project**_  :desktop_computer: 

_Project reviewed by Ikraam Ghoor (TSE)_

**Pull request merged**